edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/util/MetricsDeDuplication.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Util
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    public static class MetricsDeDuplication
+    {
+        public static IEnumerable<Metric> RemoveDuplicateMetrics(IEnumerable<Metric> metrics)
```

Comment:
Where is Metric defined? Is it generic enough to put it in Util?

Reply:
edge-agent\src\Microsoft.Azure.Devices.Edge.Agent.Diagnostics\Metric.cs

I think it makes sense to leave it in Diagnostics for now. It's only used internally